src/analysis/normed_space/star/basic.lean

```diff
@@ -178,7 +178,8 @@ nnnorm_pow_two_pow_of_self_adjoint x.property _
 
 section starₗᵢ
 
-variables [comm_semiring 𝕜] [star_ring 𝕜] [normed_ring E] [star_ring E] [normed_star_group E]
+variables [comm_semiring 𝕜] [star_ring 𝕜]
```

Comment:
Does this need commutativity?

Reply:
Yes, although it took me a minute to understand why. The reason is that this is a *conjugate* linear isometric equivalence. This is not a homomorphism of the underlying ring unless the ring is commutative (because `star (r * s) = star s * star r`). More precisely, conjugate linear maps are implicitly using `star_ring_end` (which needs commutativity) under the hood.